DXRMapFixups/DeusEx/Classes/DXRFixupVandenberg.uc

```diff
@@ -50,6 +51,12 @@ function PreFirstEntryMapFixes()
         foreach RadiusActors(class'Actor', a, 250, vectm(-4350,3000,-5850)) {
             a.Destroy();
         }
+
+        foreach AllActors(class'#var(prefix)DamageTrigger',dt){
+            if (dt.DamageType=='Shocked'){
+                dt.Tag='lab_water';
```

Comment:
do we need to do damage scaling here like we do for the paris radiation room? also probably should do the spinning room in shipfan

Reply:
I don't think this one is totally necessary, since you can jump across and disable the electricity.  This fix just made it so all the damage triggers actually get disabled by the electrical panel (From what I could see, it was just one near the computer that wasn't linked)